### PR TITLE
Seamless posting

### PR DESCRIPTION
**Pull request by SummerWish**
Create new messages seamlessly as showing the message.

Currently context-menu is broken, I'm fixing it.

### CODE REVIEW

File: javascripts/background_scripts/embeded.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/*global chrome */
```
Comment (smcgregor) [inline javascripts/background_scripts/embeded.js]: Needs file overview. Also, why is this called "embeded.js"? You previously labeled the other posting method as embeded.

I think we have been mispelling embedded for a while. embeded -> embedded

File: javascripts/background_scripts/embeded.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+/*global Privly */
+Privly.message.addListener(function (data, sendResponse) {
+  if (typeof data.action !== 'string') {
+    return;
+  }
+
+  if (data.action.indexOf('embeded/contentScript/') === 0) {
```
Comment (smcgregor) [inline javascripts/background_scripts/embeded.js]: There are several cases here and it is taking me too long to figure out what they are. You need more comments.

File: javascripts/background_scripts/modal_button.js [hunk 3]
```diff
@@ -1,64 +1,110 @@
...
+        path: {
+          19: 'images/icon_disabled_19.png',
+          38: 'images/icon_disabled_38.png',
+        }
+      });
+    }
+  }
+}
 
+function isInEmbededPosting() {
```
Comment (smcgregor) [inline javascripts/background_scripts/modal_button.js]: Comment this. Check function and @fileoverview comments everywhere.

File: javascripts/background_scripts/modal_button.js [hunk 4]
```diff
@@ -1,64 +1,110 @@
...
-      if( modalButton.badgeText === "on") {
-        modalButton.badgeText = "off";
-      } else {
-        modalButton.badgeText = "on";
-      }
-      chrome.tabs.query({currentWindow:true, highlighted: true}, readingProcess.tabsChange);
-      sendResponse();
+// Subscribe to option changed events
+Privly.message.addListener(function (request, sendRequest, sender) {
+  if (request.action === 'options/changed') {
```
Comment (smcgregor) [inline javascripts/background_scripts/modal_button.js]: This should be declared in a separate function that is added as a listener so you can document/comment it and unit test it. This is true several other places as well.

File: javascripts/content_scripts/embeded.app.js [hunk 5]
```diff
@@ -0,0 +1,367 @@
+/**
+ * @fileOverview This file handles the communication between
+ * the Privly embeded-posting app and content scripts.
+ *
```
Comment (smcgregor) [inline javascripts/content_scripts/embeded.app.js]: I thought this was the seamless posting pull request?

File: javascripts/background_scripts/embeded.js (hunk 1 above)
Comment (SummerWish) [inline javascripts/background_scripts/embeded.js]: OMG! Then I need to replace a lot of files 😂..

File: javascripts/background_scripts/embeded.js (hunk 2 above)
Comment (SummerWish) [inline javascripts/background_scripts/embeded.js]: OK

File: javascripts/content_scripts/embeded.app.js (hunk 5 above)
Comment (SummerWish) [inline javascripts/content_scripts/embeded.app.js]: ok.. I mixed seamless posting and embeded posting together.. the name is confusing
I will separate them

File: javascripts/background_scripts/embeded.js (hunk 1 above)
Comment (smcgregor) [inline javascripts/background_scripts/embeded.js]: Sorry I didn't see it sooner. Should it really be seamless.js?

Comment (smcgregor) [inline javascripts/background_scripts/embeded.js]: If not, let's use the misspelling and do a global replace once everything is reviewed and merged.

**Comment by SummerWish** [comment]
See #129

**Pull request closed without merging**